package.json

```diff
@@ -438,17 +438,22 @@
         },
         "puppet.installDirectory": {
           "type": "string",
-          "default": null,
-          "description": "The fully qualified path to the Puppet install directory. This can be a PDK or Puppet Agent installation. For example: 'C:\\Program Files\\Puppet Labs\\Puppet' or '/opt/puppetlabs/puppet'. If this is not set the extension will attempt to detect the installation directory"
+          "default": "",
+          "markdownDescription": "The fully qualified path to the Puppet install directory. This can be a PDK or Puppet Agent installation. For example: 'C:\\Program Files\\Puppet Labs\\Puppet' or '/opt/puppetlabs/puppet'. If this is not set the extension will attempt to detect the installation directory. Do **not** use when `#puppet.installType#` is set to `auto`"
```

Comment:
Does this require a minimum version bump for vscode?  I can't remember when this was added

Reply:
As stated in a top level comment, we are good to go on 1.38 which is our minimum